Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -632,16 +632,17 @@
   categories:
   - UplinkDeception
 
-- type: listing
-  id: UplinkHeadsetEncryptionKey
-  name: uplink-encryption-key-name
-  description: uplink-encryption-key-desc
-  icon: { sprite: /Textures/Objects/Devices/encryption_keys.rsi, state: synd_label }
-  productEntity: BoxEncryptionKeySyndie # Two for the price of one
-  cost:
-    Telecrystal: 2
-  categories:
-  - UplinkDeception
+# Delta-V: replaced by syndicate radio implant
```

Comment:
why not uh, drop the cost on this and keep this as an option? You can risk it all for a cheap option, or pay a premium for a hidden option?

Reply:
One person going for the cheap option obviates away the choice of everyone else. That's why the bulk purchase is still an option- you can get two for 3TC, which is only 1TC more then the original pair of keys.